Header header
string frame_id
string object_type
int32 x
int32 y
